src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -711,7 +711,7 @@ trait Definitions extends api.StandardDefinitions {
     lazy val SuccessClass = requiredClass[scala.util.Success[_]]
     lazy val FutureClass = requiredClass[scala.concurrent.Future[_]]
     lazy val PromiseClass = requiredClass[scala.concurrent.Promise[_]]
-    lazy val NonFatalClass = requiredClass[scala.util.control.NonFatal.type]
+    lazy val NonFatalClass = requiredModule[scala.util.control.NonFatal.type]
```

Comment:
```suggestion
    lazy val NonFatalModule = requiredModule[scala.util.control.NonFatal.type]
```

Reply:
that thought crossed my mind. Also I don't know how these members are/were used, and updating the forcing is extra work.